### DISCUSSION TITLE
续费后变免费版了

### DISCUSSION THREAD

**Discussion by ivykii007** [getlantern/lantern-forum-cn#526] [Q&A]
@Mike777Khoo 帮忙尽快解决一下，谢了
3月在老版本内部已购专业版，至今无法使用，已邮件反馈很多次，没得到解决！
-支付340元LANTERN专业版，日期是3月28日，交易单号是4200002140202403282606067974
请帮我解决一下不能使用问题！或者退款吧

付款时是老版本，只有第一天可以用，之后就不能用了，然后就更新了新版，也不能用。

**Comment by ivykii007** [comment]
购买后，邮箱没有收到任何确认信息。

**Reply by MaxMKMax** [reply]
@ivykii007 您好. 因为邮件被拒收了所以看不到邮件. 好的, 我帮您提交问题. 请您提供我您的账号邮箱和支付单据. 

您放心我们一定会帮您处理好. 希望您别退款. 谢谢您支持蓝灯!

**Reply by ivykii007** [reply]
![Uploading Buy info_202403.jpg…]()
ivykii2021@163.com
请查收@Mike777Khoo
![Uploading Buy info_202403.jpg…]()

**Comment by ivykii007** [comment]
![Uploading Buy info_202403.jpg…]()

**Reply by ivykii007** [reply]
能看到支付图片吗

**Comment by ivykii007** [comment]
@Mike777Khoo

**Reply by MaxMKMax** [reply]
@ivykii007 您好. 依然看不到您的图片. 也下载不了. 您可以发邮件到我的邮箱吗? 谢谢. 不好意思, 麻烦您了.
mike@getlantern.org

**Comment by ivykii007** [comment]
Dear Mike

看下附件支付信息是否可以打开了？

***@***.******@***.***都帮我看下。

谢谢




在 2024-05-27 00:42:28，"Mike K" ***@***.***> 写道：

@ivykii007 您好. 依然看不到您的图片. 也下载不了. 您可以发邮件到我的邮箱吗? 谢谢. 不好意思, 麻烦您了.
***@***.***

**Reply by MaxMKMax** [reply]
@ivykii007 您好 Ivy. 依然看不见呢. 您方便发个邮件给我吗? 附在邮件上或许可以. 要不然就放Google drive给我. 分享查看权限. 谢谢!

mike@getlantern.org

**Comment by ivykii007** [comment]
Mike，这次看得到支付信息吗？附件和插入邮件正文都有一份。




在 2024-05-27 20:19:00，"Mike K" ***@***.***> 写道：

@ivykii007 您好 Ivy. 依然看不见呢. 您方便发个邮件给我吗? 附在邮件上或许可以. 要不然就放Google drive给我. 分享查看权限. 谢谢!

***@***.***

**Reply by MaxMKMax** [reply]
@ivykii007 嗯嗯. 看到了. 已经帮您提交问题好了. 请您给我们几天工作日完成. 谢谢.

**Comment by ivykii007** [comment]
Hi Mike，情况如何了？




在 2024-05-28 00:59:15，"Mike K" ***@***.***> 写道：

@ivykii007 嗯嗯. 看到了. 已经帮您提交问题好了. 请您给我们几天工作日完成. 谢谢.

**Reply by MaxMKMax** [reply]
@ivykii007 您好. 不好意思让您久等了. 我们的负责人这几天不在线. 等他回来我马上帮您解决问题. 很抱歉.

**Comment by ivykii007** [comment]
Mike，还没好吗？



---- 回复的原邮件 ----
| 发件人 | Mike ***@***.***> |
| 日期 | 2024年06月07日 00:55 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [getlantern/lantern-forum-cn] 续费后变免费版了 (Discussion #526) |

@ivykii007 您好. 不好意思让您久等了. 我们的负责人这几天不在线. 等他回来我马上帮您解决问题. 很抱歉.

**Reply by MaxMKMax** [reply]
@ivykii007 刚刚解决了. 您查看是否成功恢复专业版.

**Comment by ivykii007** [comment]
***@***.***授权专业版，收不到验证码。







在 2024-06-19 01:15:45，"Mike K" ***@***.***> 写道：

@ivykii007 刚刚解决了. 您查看是否成功恢复专业版.

**Comment by warm3snow** [comment]
问一下修复好了没？或者退款了没，碰到相同的问题。

**Reply by MaxMKMax** [reply]
@warm3snow https://lantern.surveysparrow.com/s/propayment-triage-form/tt-ZxBeJ
您好. 为了保护您的隐私. 请您发支付单据和账号邮箱到这个链接. 谢谢.

**Comment by ivykii007** [comment]
已发送，请查收！多久可以好？

在 2024-06-21 02:42:19，"Mike K" ***@***.***> 写道：


@warm3snowhttps://lantern.surveysparrow.com/s/propayment-triage-form/tt-ZxBeJ
您好. 为了保护您的隐私. 请您发支付单据和账号邮箱到这个链接. 谢谢.
